app/src/main/java/org/mifos/selfserviceapp/ui/adapters/SavingAccountsListAdapter.java

```diff
@@ -104,6 +119,15 @@ public int getItemCount() {
         @BindView(R.id.iv_status_indicator)
         CircularImageView iv_status_indicator;
 
+        @BindView(R.id.ll_account_detail)
+        LinearLayout ll_AccountDetail;
```

Comment:
use a proper naming convention like ``llAccountDetail`` and change below variables too